packages/eslint-plugin/src/rules/prefer-nullish-coalescing.ts

```diff
@@ -139,8 +140,8 @@ export default createRule<Options, MessageIds>({
     ) {
       context.report({
         loc: {
-          start: { line: 0, column: 0 },
-          end: { line: 0, column: 0 },
+          end: { column: 0, line: 0 },
```

Comment:
oof. 
you know how i'll feel about this 😆

Reply:
Oh, actually - I don't really care about the line vs column thing, but it's the `{ end, start }` that grinds my gears

Reply:
what is time...